api/src/test/java/com/flipkart/flux/api/WorkflowStateSummaryTest.java

```diff
@@ -10,24 +10,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.flipkart.flux.commons.dto;
+package com.flipkart.flux.api;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.IOUtils;
+import org.hamcrest.core.Is;
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
```

Comment:
1. Why do you say it is good practice ? :)
2. It doesn't have any performance impact in any class - even if its a production class